integration-tests/common/test_common.go

```diff
@@ -21,7 +21,7 @@ import (
 	test_env_ctf "github.com/smartcontractkit/chainlink-testing-framework/lib/docker/test_env"
 	"github.com/smartcontractkit/chainlink-testing-framework/lib/utils/testcontext"
 
-	"github.com/smartcontractkit/chainlink/integration-tests/client"
+	client "github.com/smartcontractkit/chainlink/deployment/environment/nodeclient"
```

Comment:
Is this change intentional?

Reply:
Yes, it was necessary in order to fix errors resulting from updating the ref to `chainlink/integration-tests` in `chainlink-solana/integration-tests/go.mod`. The `client` package was moved into a new `deployment` package in recent versions of `chainlink`, but nobody had updated the Solana repo with that change for some reason.